test/parallel/test-global-console-exists.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const assert = require('assert');
+const events = require('events');
+
+const old_default = events.defaultMaxListeners;
+events.defaultMaxListeners = 1;
+
+const e = new events.EventEmitter();
+e.on('hello', function() {});
+
+assert.ok(!e._events['hello'].hasOwnProperty('warned'));
```

Comment:
The `listeners()` API leaves out the `warned` property: https://github.com/nodejs/node/blob/master/lib/events.js#L393.